test/extended/prometheus/prometheus.go

```diff
@@ -159,13 +201,50 @@ var _ = g.Describe("[sig-instrumentation][Late] OpenShift alerting rules [apigro
 				violations.Insert(
 					fmt.Sprintf("WARNING: Alert %q is critical and has no 'runbook_url' annotation", alert.Name),
 				)
-			} else if runbook != "" {
-				// If there's a 'runbook_url' annotation, make sure it's a
-				// valid URL and that we can fetch the contents.
-				if err := helper.ValidateURL(runbook, 10*time.Second); err != nil {
+			}
+
+			return violations
+		})
+
+		if err != nil {
+			e2e.Failf(err.Error())
+		}
+	})
+
+	g.It("should have valid runbook URLs", func() {
+		err := helper.ForEachAlertingRule(alertingRules, func(alert promv1.AlertingRule) sets.String {
+			violations := sets.NewString()
+			runbook_url := string(alert.Annotations["runbook_url"])
+
+			if runbook_url != "" {
```

Comment:
(nit) easier to read.
```suggestion
			if runbook_url == "" {
				return nil
			}
```